lib/fizzy/execute.cpp

```diff
@@ -687,7 +688,8 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             const auto br_table_size = read<uint32_t>(immediates);
             const auto arity = read<uint32_t>(immediates);
 
-            const auto br_table_idx = stack.pop();
+            // TODO: is uint64_t the correct stack type here?
+            const auto br_table_idx = stack.pop().as<uint64_t>();
```

Comment:
@gumb0 do you know this top of your head: what is the idx type for `br_table`?

Reply:
We have a nice table to look it up :)
https://github.com/wasmx/fizzy/blob/0728606b601960b51177f3fa7ed7cb2ef3e6110f/lib/fizzy/instructions.cpp#L29

Reply:
Here's the validation spec that defines input value as `i32`
https://webassembly.github.io/spec/core/valid/instructions.html#xref-syntax-instructions-syntax-instr-control-mathsf-br-table-l-ast-l-n